### DISCUSSION TITLE
Graph contains only bidirected edges

### DISCUSSION THREAD

**Discussion by juliusd01** [jakobrunge/tigramite#360] [Q&A] [answered]
Hello,
I used PCMCI to learn causal connections on a self-created timeseries dataset. The resulting graph only contains bidirected edges (also for another dataset with more nodes) :

![graph_with_only_bidirectional_edges](https://github.com/jakobrunge/tigramite/assets/107621692/04136c4e-a491-4d58-9915-ab5723a56a64)

The data contained events, where a 1 means a fault is active. When converting to timeseries, there was also some noise added to not only get 0s and 1s. This would be a very small example of how the data looks like:

| Error1   | Error2   | Error 4
|----------|----------|--------|
| 1.00034  | 0.00078  | 0.00084
| 1.00018  | 0.00056  | 1.00023

Does anybody know what properties of the dataset could be the cause that PCMCI only detects bidirectional edges?

**Comment by jakobrunge** [comment, marked as answer]
Please check the tutorials, these are lagged directed edges in both directions. Bidirected edges mean "confounded" in Tigramite's convention. Also check the tutorials for the appropriate independence test to use. Event data can only be handled as categorical data with Tigramite. Then use the Gsquared, CMIsymb, or for mixed data the RegressionCI test.
